setup.py

```diff
@@ -118,7 +127,7 @@ def run(self):
     author="Red Hat, Inc.",
     author_email="info@ansible.com",
     url="https://github.com/ansible/galaxy_ng/",
-    python_requires=">=3.8,<3.9",
+    python_requires=">=3.8",
```

Comment:
may need to rebase as i think this change has already been committed to `master`

Reply:
oopsie. Didn't mean to commit that